tests/test_utils.py

```diff
@@ -158,12 +158,24 @@ def test_pil_version(self):
             pil_image = Image.open(assets.path_to('grenzboten-test/data/OCR-D-IMG-BIN/p179470.tif'))
             pil_image.crop(box=[1539, 202, 1626, 271])
 
-    def test_pushd_popd(self):
+    def test_pushd_popd_newcwd(self):
         cwd = getcwd()
         with pushd_popd('/tmp'):
             self.assertEqual(getcwd(), '/tmp')
         self.assertEqual(getcwd(), cwd)
 
+    def test_pushd_popd_tempdir(self):
+        cwd = getcwd()
+        with pushd_popd(tempdir=True) as newcwd:
+            self.assertEqual(getcwd(), newcwd)
+            self.assertTrue(newcwd.startswith('/tmp'))
```

Comment:
Is this reliable across platforms? (I thought even on Unix Python must first try `TMPDIR`, `TEMP` and `TMP` first.)

Reply:
b74c5adf